v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -0,0 +1,93 @@
+from enum import Enum
+from types import MappingProxyType
+
+from v03_pipeline.lib.model import AccessControl, DatasetType, Env, ReferenceGenome
+from v03_pipeline.lib.reference_datasets.cadd import load_cadd_ht_from_raw_dataset
+from v03_pipeline.lib.reference_datasets.hgmd import (
+    HGMD_ENUM_SELECT,
+    download_and_import_hgmd_vcf,
+)
+
+LOAD_DATASET_FUNC = 'load_parsed_dataset_func'
+VERSION = 'version'
+RAW_DATASET_PATH = 'raw_dataset_path'
+ENUM_SELECT = 'enum_select'
+
+
+class ReferenceDataset(str, Enum):
+    cadd = 'cadd'
+    hgmd = 'hgmd'
+
+    CONFIG = MappingProxyType(
+        {
+            'cadd': {
+                LOAD_DATASET_FUNC: load_cadd_ht_from_raw_dataset,
+                ReferenceGenome.GRCh37: {
+                    VERSION: '1.0',
+                    RAW_DATASET_PATH: [
+                        'https://krishna.gs.washington.edu/download/CADD/v1.7/GRCh37/whole_genome_SNVs.tsv.gz',
+                        'https://krishna.gs.washington.edu/download/CADD/v1.7/GRCh38/gnomad.genomes.r4.0.indel.tsv.gz',
+                    ],
+                },
+                ReferenceGenome.GRCh38: {
+                    VERSION: '1.0',
+                    RAW_DATASET_PATH: [
+                        'https://krishna.gs.washington.edu/download/CADD/v1.7/GRCh38/whole_genome_SNVs.tsv.gz',
+                        'https://krishna.gs.washington.edu/download/CADD/v1.7/GRCh37/gnomad.genomes-exomes.r4.0.indel.tsv.gz',
+                    ],
+                },
+            },
+            'hgmd': {
+                LOAD_DATASET_FUNC: download_and_import_hgmd_vcf,
+                ReferenceGenome.GRCh37: {
+                    VERSION: '1.0',
+                    RAW_DATASET_PATH: 'gs://seqr-reference-data/v3.1/GRCh37/reference_datasets/hgmd-v2020.1.vcf.bgz',
```

Comment:
HGMD can never be in the public bucket. This needs to be removed immediately

Reply:
I don't think this is actually there, it's just a typo!

Reply:
😌

Reply:
also I do not think we want to use an HGMD version from 2020, the current config is using 2023

Reply:
I think copilot autocompleted this path and I didn't check. It's not actually in there